meta-ov/recipes-apps/ovmenu-ng-skripts/files/upload-all.sh

```diff
@@ -2,13 +2,10 @@
 
 USB_PATH="/usb/usbstick/openvario/upload"
 UPLOAD_PATH="/home/root/.xcsoar"
-if [ -z "$(ls $USB_PATH/* 2>/dev/null)" ]; then
+if [ -z "$(ls ${USB_PATH}/* 2>/dev/null)" ]; then
```

Comment:
You add curly braces (why??), but the parameter is still unquoted; everywhere else, expanded variables are quoted.